### ISSUE TITLE
[My health] Error messages

### ISSUE THREAD

**Issue by laurwill** [department-of-veterans-affairs/va.gov-team#96184]

**Comment by laurwill** [comment]
@sterkenburgsara I made a stub, want to fill in whatever info here and I'll take a pass when you say it's ready?

**Comment by sterkenburgsara** [comment]
@laurwill thank you! updated, I'll poke you about this when you're back from PTO.

**Comment by sterkenburgsara** [comment]
I don't think we need anything else - this work is fully implemented within My HealtheVet on the web & the VAHB mobile app team is aware of the need to implement similar alerts on their end. I believe we can close!

**Comment by mistymg** [comment]
Awesome, thanks @sterkenburgsara. Closing out.
